index.html

```diff
@@ -4,8 +4,8 @@
   <head>
     <title>Lambda School Newsfeed</title>
     <link rel="stylesheet" type="text/css" media="screen" href="./CSS/index.css" />
-    <script src="./Menu/Menu.js"></script>
-    <script src="./Article/Article.js"></script>
+    <script src="./Menu/Menu.js" defer></script>
```

Comment:
I would suggest using async as a default unless you have a specific use case for using defer.